Source/EasyNetQ.Management.Client/IManagementClient.cs

```diff
@@ -702,4 +702,18 @@ public interface IManagementClient : IDisposable
     /// <param name="cancellationToken"></param>
     /// <returns></returns>
     Task RebalanceQueuesAsync(CancellationToken cancellationToken = default);
+
+    /// <summary>
+    ///     Creates a shovel in a specific vhost
+    /// </summary>
+    /// <param name="vHostName"></param>
+    /// <param name="name"></param>
+    /// <param name="parameters"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task CreateShovelAsync(
+        string vHostName,
```

Comment:
Please rename this parameter to vhostName as it is named in other methods.

Reply:
Ok, renamed `vHostName` to `vhostName`